stacker_blueprints/iam_roles.py

```diff
@@ -73,17 +44,21 @@ def generate_policy_statements(self):
 
         return []
 
-    def create_policy(self, name):
+    def create_policy(self, name=None):
         statements = self.generate_policy_statements()
         if not statements:
             return
 
         t = self.template
 
+        logical_name = "Policy"
+        if name:
+            logical_name += name
```

Comment:
This reverses the name of of the policy from the blueprint before right?  Should probably make it match the way it was before so we don't have people's stacks changing out of the blue.

Reply:
Thanks! Didn't mean to change the order.